authors/apps/articles/tests/test_article_model.py

```diff
@@ -0,0 +1,49 @@
+from django.test import TestCase
+
+from authors.apps.articles.models import Article
+from authors.apps.authentication.models import User
+
+
+class ArticleTestCase(TestCase):
+    """ This class defines the test suite for the article model """
+
+    def setUp(self):
```

Comment:
There are tests that have similar creations of users in the authentication folder.
Please put all data in setup methods in one base class that is inherited by all current and newly created test classes.